mozregression/utils.py

```diff
@@ -60,6 +61,15 @@ def names(self):
 CACHE_SESSION = None
 
 
+def retry_get(**kwargs):
```

Comment:
So, basically I have to return redo.retry(get_http_session().get, attempts=3, sleeptime=1, retry_exceptions=(..., ...), args=(url), kwargs=kwargs), right?